graph_node_nft_metrics.py

```diff
@@ -0,0 +1,83 @@
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+from metrics import *
+
+import logging
+logger = logging.getLogger(__name__)
+logging.getLogger("gql").setLevel(logging.WARNING)
+
+
+def connect_graph_node(graph_node_url):
+    """Connect to the Graph Node using the provided URL."""
+    try:
+        transport = RequestsHTTPTransport(
+            url=graph_node_url,
+            verify=True,
+            retries=3,
+        )
+        client = Client(transport=transport, fetch_schema_from_transport=True)
+        logger.info(f"Successfully connected to Graph Node: {graph_node_url}")
+        return client
+    except Exception as e:
+        logger.error(f"Failed to connect to Graph Node: {e}")
+        raise
+
+
+def get_latest_block(client):
+    """Get latest block number"""
+    try:
+        query = gql('''
+        query {
+            _meta {
+                block {
+                    number
+                }
+            }
+        }
+        ''')
+        response = client.execute(query)
+        block = response['_meta']['block']['number']
+        FLUENCE_SUBGRAPH_LATEST_BLOCK.set(block)
```

Comment:
This should be a separate metric or subgraph's URI should be a label. 

Otherwise you're mixing up two subgraph's last blocks in a single metric.